i18n/nl.json

```
@@ -14,10 +14,10 @@
     "bounces.unknownService": "Onbekende service.",
     "bounces.view": "Zie bounces",
     "campaigns.addAltText": "Voeg alternatieve tekst zonder opmaak toe",
-    "campaigns.archive": "Archive",
-    "campaigns.archiveEnable": "Publish to public archive",
-    "campaigns.archiveHelp": "Publish (running, paused, finished) the campaign message on the public archive.",
-    "campaigns.archiveMeta": "Campaign metadata",
+    "campaigns.archive": "Archiveren",
+    "campaigns.archiveEnable": "Publiceren naar publiek archief.",
```

There's no trailing period in the original English text here.

```suggestion
    "campaigns.archiveEnable": "Publiceren naar publiek archief",
```